test/known_issues/test-permission-model-path-resolution.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+// The permission model resolves paths to avoid path traversals, but in doing so
+// it potentially interprets paths differently than the operating system would.
+// This test demonstrates that merely enabling the permission model causes the
+// application to potentially access a different file than it would without the
+// permission model.
+
+const common = require('../common');
+// TODO(tniessen): figure out why this doesn't work on Windows
+if (common.isWindows || !common.canCreateSymLink())
+  common.skip('insufficient privileges');
```

Comment:
```suggestion
  common.skip('insufficient privileges');
  require('assert').fail('insufficient privileges')
```

Since it's a `known_issue`, we call assert.fail when skipping a test/platform.

Reply:
Ah, that's unfortunate, but makes sense. Thanks!